CMakeLists.txt

```diff
@@ -0,0 +1,254 @@
+cmake_minimum_required(VERSION 3.0.2)
+set(CMP0048 NEW)
+
+project(_tango LANGUAGES CXX
+        VERSION 1.0)
+        
+#need to define the version of the library
+set(MAJOR_VERSION "9")
+set(MINOR_VERSION "3")
+set(PATCH_VERSION "1")
+
+#convenient versions
+set(LIBRARY_VERSION "${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}")
+if(NOT WIN32)
```

Comment:
Agree

Reply:
Are you saying that the whole CMake effort was only for supporting Windows? I wonder what is missing to support Linux...

Reply:
On linux we don't need cmake. A simple `python setup.py build` works well.

I propose that if you are interested to open another PR to make these improvements after otherwise the work will never end :-)

To be completely honest I don't think we need cmake on windows neither.